tests_torch/test_differentiation.py

```diff
@@ -1,25 +1,30 @@
 import unittest
 
 import numpy as np
+import torch
 
 import neural_renderer_torch
 
 
 class TestDifferentiation(unittest.TestCase):
     def test_backward(self):
         images = np.random.normal(size=(10, 32, 32, 3)).astype('float32')
+
         x = np.tile(np.arange(32).astype('float32')[None, None, :, None], (10, 32, 1, 1))
         y = np.tile(np.arange(32).astype('float32')[None, :, None, None], (10, 1, 32, 1))
+
         coordinates = np.concatenate((x, y), axis=-1)
         coordinates = ((coordinates / 31) * 2 - 1) * 31. / 32.
+
         noise = np.random.normal(size=(10, 32, 32, 3)).astype('float32')
         step = 2 / 32.
 
-        images = chainer.cuda.to_gpu(images)
-        coordinates = chainer.Variable(chainer.cuda.to_gpu(coordinates))
-        noise = chainer.cuda.to_gpu(noise)
+        device = torch.device('cuda:0')
```

Comment:
It may be better to use `to_gpu` function within utils. Here is a sample usage. https://github.com/dBeker/neural_renderer_v2_pytorch/blob/master/tests_torch/test_rasterize.py#L34